.github/workflows/build-web-extension.yml

```diff
@@ -0,0 +1,45 @@
+name: Build Web Extension
+
+on:
+    workflow_dispatch:
+
+    push:
+        branches:
+            - "v4"
+            - "master"
+        paths:
```

Comment:
This makes it so that the automatic building only happens if any file in these paths changes. If you change something for the `cordova` or `server` packages, for example, there's no need to build the web extension again.

Reply:
If we trigger builds only on explicit releases/tags (as mentioned above), all targets should be rebuilt with a version bump regardless of whether they have changed or not. This gives us consistent versioning across targets and people only need to refer to the latest release to find whatever they're looking for.

Reply:
Thanks for sharing your thought process here. As I mentioned above, I think it's best to keep this for the "continuous" part of CI/CD, and have a separate action (ideally just manual, for now, but can also be automatic, on tags) to release, which can do the process for all targets.

Let me know if you agree.